main.go

```diff
@@ -65,54 +67,45 @@ func main() {
 		Desc:   "Maximum records to be queried to TME",
 		EnvVar: "MAX_RECORDS",
 	})
-	batchSize := app.Int(cli.IntOpt{
-		Name:   "batchSize",
+	slices := app.Int(cli.IntOpt{
+		Name:   "slices",
 		Value:  int(10),
 		Desc:   "Number of requests to be executed in parallel to TME",
-		EnvVar: "BATCH_SIZE",
-	})
-	cacheFileName := app.String(cli.StringOpt{
-		Name:   "cache-file-name",
-		Value:  "cache.db",
-		Desc:   "Cache file name",
-		EnvVar: "CACHE_FILE_NAME",
+		EnvVar: "SLICES",
 	})
 
-	tmeTaxonomyName := "topics"
+	tmeTaxonomyName := "series"
```

Comment:
Hi - no it's not; there isn't a taxonomy for series setup yet in TME (as far as I know), so this is just a placeholder.